inc/api.php

```diff
@@ -59,7 +59,7 @@ function gwapi_send_sms($message, $recipients, $sender='', $destaddr='MOBILE')
 
     foreach($recipients_formatted as $msisdn => $tags) {
         $rec = [
-            'msisdn' => (int)$msisdn,
+            'msisdn' => gwapi_bigint($msisdn),
```

Comment:
It does, and we will keep this behavior for the same reasons, although the MSISDN does not include the + prefix, this is a very common mistake so we will allow it.